packages/teams-js/src/internal/telemetry.ts

```diff
@@ -202,6 +202,8 @@ export enum ApiName {
   Navigation_NavigateToTab = 'navigation.navigateToTab',
   Navigation_ReturnFocus = 'navigation.returnFocus',
   Notifications_ShowNotification = 'notifications.showNotification',
+  OtherAppStateChange_Install = 'otherApp.install',
+  OtherAppStateChange_UnregisterInstall = 'otherApp.unregisterInstall',
```

Comment:
The lack of symmetry here seems notable; intentional?

"install" vs "unregisterInstall"

Should it be "registerInstall" vs "unregisterInstall"?

Reply:
This is a great question! Let me explain my reasoning.

This is where the `OtherAppStateChange_Install` telemetry string is used in `otherAppStateChange.ts`:

![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/18446904/1a5f3f19-90f4-4a05-9d62-e4a7922741df)

The two instances of this string are used for different things:

The first line, all we really need there is a string that uniquely identifies this function call from all other function calls we are tracking. That way we can track whether or not functions are getting used by developers. This string would, you are correct, be more correctly written out to be something like `'otherApp.registerInstall'`

The second line though needs to be the name of the event that this function will receive from the host. So if the function registers for install events, the second string needs to be `'otherApp.install'` because we are receiving this event every time one or more apps are installed.

This leaves us with two options.

Option 1: Make two separate string literals, 'otherApp.registerInstall' and 'otherApp.install'. The code is easier to read but we now have two strings we need to keep in our minified js and devs have to pay the cost of downloading both.

Option 2: Use the same string for both. The code is a little more confusing but we only make one string literal.

Ultimately, I thought it was keep the package a little smaller which is why I landed on this solution. Good catch, and hopefully my explanation makes sense. Let me know if you disagree or want me to change it.